### ISSUE TITLE
Add a method to get learning rate schedule from restored network.

### ISSUE THREAD

**Issue by baedaron** [deeplearning4j/deeplearning4j#9798]
or is there a way to get it?

i need to set a custom ISchedule and manipulate it when my neural network is restored.

**Comment by agibsonccc** [comment]
@baedaron there's not 1 "schedule" it's actually per layer. Could you clarify what you need to do exactly? I've never heard of anyone needing to do that. We have methods to set the learning rate if that's what you need. You can see that here: https://github.com/eclipse/deeplearning4j/blob/f0504431bb6247f4bd0ca27d1c0a7047d50dce15/deeplearning4j/deeplearning4j-nn/src/main/java/org/deeplearning4j/util/NetworkUtils.java#L235 - either what whatever you want will be per layer.
